Packages/Newtonsoft.Json-for-Unity.Converters/UnityConverters/Geometry/BoundsIntConverter.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+/*WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW*\     (   (     ) )
+|/                                                      \|       )  )   _((_
+||  (c) Wanzyee Studio  < wanzyeestudio.blogspot.com >  ||      ( (    |_ _ |=n
+|\                                                      /|   _____))   | !  ] U
+\.ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ./  (_(__(S)   |___*/
+
+using UnityEngine;
+using UnityEngine.Scripting;
+
+namespace Newtonsoft.Json.UnityConverters.Geometry
+{
+    /// <summary>
+    /// Custom Newtonsoft.Json converter <see cref="JsonConverter"/> for the Unity Bounds type <see cref="Bounds"/>.
+    /// </summary>
+    public class BoundsIntConverter : PartialVector3IntConverter<BoundsInt>
+    {
+        private static readonly string[] _memberNames = { "position", "size" };
+
+        public BoundsIntConverter()
+            : base(_memberNames)
+        {
+        }
+
+        /// <summary>
+        /// Prevent the properties from being stripped.
+        /// </summary>
+        [Preserve]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Code Quality", "IDE0051:Remove unused private members", Justification = "Ensures the properties are preserved, instead of adding a link.xml file.")]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 206 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461512)